tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1212,6 +1243,27 @@ static uint8_t set_filter_list(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_SUCCESS;
 }
 
+static uint8_t set_extended_advertising(const void *cmd, uint16_t cmd_len,
+					void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_gap_set_extended_advertising_cmd *cp = cmd;
+	struct btp_gap_set_extended_advertising_rp *rp = rsp;
+
+	LOG_DBG("ext adv settings: %d", cp->settings);
+
+	if (cp->settings & 0x01) {
```

Comment:
Is `cp->settings` a bitfield? If so, then I suggestion that we have an `enum` or `#define`s for each possible bit. 

If this is just a `if (cp->settings != 0)` check, then I suggest to make that more clear and avoid the `& 0x01` check

Reply:
This is defined in the BTP documentation as if it could be a bit field, but for now only the first bit is used. So let's use `if (cp->settings != 0)` for now.